raster/r.sim/simlib/waterglobs.h

```diff
@@ -6,6 +6,10 @@
 #define UNDEF	-9999
 
 #include <grass/raster.h>
+#ifdef _MSC_VER
+#undef min
+#undef max
+#endif
```

Comment:
Recently, r.sim got major changes in #5373 and this file and global variables have been removed.